src/main/app/jurisdiction/JurisdictionPostController.ts

```diff
@@ -20,4 +23,30 @@ export class JurisdictionPostController extends PostController<AnyObject> {
     req.body.connections = addConnection({ ...req.session.userCase, ...formData });
     await super.post(req, res);
   }
+
+  protected async save(req: AppRequest<AnyObject>, formData: Partial<Case>, eventName: string): Promise<CaseWithId> {
+    const unreachableAnswersAsNull = getJurisdictionUnreachableAnswersAsNull(req.session.userCase);
+    const dataToSave = {
+      ...unreachableAnswersAsNull,
+      ...formData,
+    };
+
+    return req.locals.api.triggerEvent(req.session.userCase.id, dataToSave, eventName);
+  }
 }
+
+const getJurisdictionUnreachableAnswersAsNull = (userCase: Partial<Case>) => {
+  const jurisdictionFields = [
+    'applicant1DomicileInEnglandWales',
+    'applicant2DomicileInEnglandWales',
+    'bothLastHabituallyResident',
+    'applicant1LivingInEnglandWalesTwelveMonths',
+    'applicant1LivingInEnglandWalesSixMonths',
+    'jurisdictionResidualEligible',
+  ];
+
+  const possibleAnswers = getAllPossibleAnswersForPath(userCase, stepsWithContentApplicant1);
+  jurisdictionFields.forEach(field => (!possibleAnswers.includes(field) ? (userCase[field] = null) : userCase[field]));
```

Comment:
that else statement is not really doing anything and it kind of feels like dancing around using a for statement. I would probably be inclined to go for 

```
for (const field of jurisdictionFields) {
  if (!possibleAnswers.includes(field)) {
    userCase[field] = null;
  }
}
```

but equally you could use reduce if you want to do some fancy functional stuff.